### DISCUSSION TITLE
Few-shot, zero-shot 등 category 분류

### DISCUSSION THREAD

**Discussion by Kyumin-Park** [yukyunglee/Awesome-Dialogue-State-Tracking#10] [General]
1. Few-shot 등 카테고리를 굳이 데이터셋 밑에 둘 필요가 없어보이는데 fewshot/zeroshot이랑 etc를 다른 대분류로 빼는거 어떻게 생각하시나요?
2. Multiwoz 밑에 ontology-based, open-vocab based를 나누는게 나아보이는지 의견 궁금합니다. 
3. DSTC를 따로 뺀다는건 아래 예시의 위와 같은 분류를 아래로 내리는거라 보면 될까요..? 그렇게 되더라도 사실상 추가되는 페이퍼들이 DSTC8-SGD에 많이 몰릴꺼같긴 한데, 구도가 괜찮을것같으면 상관은 없을꺼같아요

# 4. SGD
- - - 
# Competition
## DSTC8-SGD (AAAI 20)

**Comment by yukyunglee** [comment]
Competition에 DSTC를 따로 분류하고 회차를 안쓰려 했었어요 ! 아직제가 DSTC8 페이퍼들을 안찾아봐서 모르겠습니다 ㅠㅠ 
요건 페이퍼 더 찾아보고 2차 회의때 가지치기 하시죠!
(그리고 SGD 데이터 관련해서 논문이 두개 있더라구요 ! 제가 다른 디스커션에 올려뒀으니 참고 해주시면 됩니댜)
